README.md

```diff
@@ -199,3 +269,90 @@ format is CycloneDX so the `-f` isn't needed)
 ``` bash
 bomctl export <UUID> -o merged-sbom.cdx.json
 ```
+
+### SBOM Alias
+
+Working with Multiple SBOMs, it can be tricky to keep documents straight, as different formats have different expectations for naming. `bomctl`  
+typically relies on UUIDs for storing documents, however these can be taxing to use as an ID when performing operations. The `alias` command  
+gives the option to pick a more human readable name for a document for ease of use.
+
+To get more information about `alias` and its' subcommands, run:
+
+```bash
+bomctl alias --help
+```
+
+#### __Step 1: Set Alias__
+
+We're going to use the cdx SBOM fetched in the first [exercise](#step-1-fetch). If you haven't already, go back and fetch it.
+
+```bash
+bomctl alias set urn:uuid:f360ad8b-dc41-4256-afed-337a04dff5db 0_3_0
+```
+
+#### __Step 2: Remove Alias__
+
+This commmand removes the existing alias for the provided SBOM ID.
+
+```bash
+bomctl alias remove urn:uuid:f360ad8b-dc41-4256-afed-337a04dff5db
+```
+
+### SBOM Tags
+
+Working with Multiple SBOMs, it can be hard to keep documents organized. There may be groups of SBOMs within your system that are related and should  
+be handled together, or at least linked in a way that makes it easy to know what goes with what. The `tag` command allows a user to specifcy a 'tag'  
+to associate with an SBOM to make searching for and organizing your SBOMs easier.
+
+To get more information about `tag` and its' subcommands, run:
+
+```bash
+bomctl tag --help
+```
+
+#### __Step 1: Add Tag__
+
+We're going to use the cdx SBOM fetched in the first [exercise](#step-1-fetch). If you haven't already, go back and fetch it.  
+Feel free to use any already in your Database instead, just replace the SBOM ID in the command with your chosen SBOM ID.
+
+Since this SBOM originated from a bomctl container image, lets add those tags. Feel free to add more, space separated.
```

Comment:
```suggestion
Since this SBOM originated from a bomctl container image, lets add those tags. Additional SBOM IDs can be specified separated by spaces.
```